app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      include Wor::Paginate
+      before_action :authenticate_user!
+      def index
+        render_paginated Book
```

Comment:
You install serializer gem but you are not serializing the records

Reply:
Done